instrumentation/net/http/otelhttp/transport.go

```diff
@@ -102,6 +105,10 @@ func (t *Transport) RoundTrip(r *http.Request) (*http.Response, error) {
 
 	ctx, span := tracer.Start(r.Context(), t.spanNameFormatter("", r), opts...)
 
+	if t.clientTrace != nil {
+		ctx = httptrace.WithClientTrace(ctx, t.clientTrace(ctx))
+	}
```

Comment:
Just out of curiosity, would having a default client trace be helpful? would that be considered an API breakage at this point?

Reply:
I don't think it would be API breaking, but perhaps a surprising change in behavior.  I'm not sure a default client trace is a good idea, either, as it adds overhead and verbosity that is not necessary in many cases.

Reply:
Makes sense, thanks. Very helpful, as always!